snow/engine/snowman/engine.go

```diff
@@ -431,6 +432,10 @@ func (e *Engine) QueryFailed(ctx context.Context, nodeID ids.NodeID, requestID u
 	return e.executeDeferredWork(ctx)
 }
 
+func (*Engine) SimplexMessage(_ context.Context, _ ids.NodeID, _ *p2p.Simplex) error {
```

Comment:
I am unsure we should be returning an error here. 

What if a malicious node sends us a Simplex message? In that case we will error and halt the chain.